.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: java
+jdk:
+  - oraclejdk8
+script: "mvn verify"
```

Comment:
It's because of the `maven-failsafe-plugin` indeed.

The plugin is bound to the goals `integration-test` and `verify`

The required `jetty-server` for these tests:
- starts during `pre-integration-test`
- stops during `post-integration-test`

If you have a better solution or preferred alternative, please let me know so i can adjust accordingly.